contracts/Front/Challenges.sol

```diff
@@ -79,12 +78,11 @@ contract Challenges is Base {
             uploaderId,
             bundleId,
             0x0,
-            msg.value / challengesCount,
+            (msg.value).div(challengesCount),
             time.currentTimestamp(),
             challengesCount,
             challengesStore.getNextChallengeSequenceNumber()
```

Comment:
I would argue that it no longer makes sense to pass the sequenceNumber from here. Just asking the challenges store about something just to pass to it it as a parameter is somewhat weird.

Reply:
It's very useful for cooldown testing.

Reply:
Making all methods and variables public also helps with testing. We still don't do this 😝 

The more I think about this, the more I'm convinced this parameter should go. And BTW: the `incrementNextChallengeSequenceNumber` should be made private too.

Reply:
Then any tips how to go through testing it smoothly?